### ISSUE TITLE
Support for RF-DETR Seg-Preview

### ISSUE THREAD

**Issue by StefanNa3Shape** [CVHub520/X-AnyLabeling#1184]
### Search before asking

- [x] I have searched the X-AnyLabeling [issues](https://github.com/CVHub520/X-AnyLabeling/issues) and found no similar feature requests.

### Description

Hi, Could you add RF-DETR Seg-Preview to your models?
I would like to use it for annotating.
If you point me to where, I can try help develop what is needed.

### Are you willing to submit a PR?

- [x] Yes I'd like to help by submitting a PR!

**Comment by CVHub520** [comment]
Hello @StefanNa3Shape,

Thanks for reaching out and for your interest in RF-DETR Seg-Preview! 😊

Great news – RF-DETR Seg-Preview is already supported in X-AnyLabeling. You should be able to find it available in the latest commit for your annotation tasks.

We really appreciate your enthusiasm and willingness to help by submitting a PR! While this specific model is already integrated, your offer to contribute is always valued. If you have any further suggestions or encounter any issues, please don't hesitate to open another issue or discussion.

Happy labeling! 🚀

**Comment by Tensing2009** [comment]
That is amazing news!
But I have trouble importing my model.
Could you help?


I exported it as follows after training:
```
from rfdetr import RFDETRSegPreview
import torch
checkpoint = r".\runs\coco_merged\RFDETRSegPreview432\checkpoint.pth"
device = "cuda" if torch.cuda.is_available() else "cpu"
rfdetr_model = RFDETRSegPreview(pretrain_weights=checkpoint, device=device,resolution=576)
rfdetr_model.export()
```

Then I made the model yaml file as follows (model and .yaml are in the same directory):
according to https://github.com/CVHub520/X-AnyLabeling/blob/main/anylabeling/configs/auto_labeling/rfdetr_seg_preview.yaml

```
type: rfdetr_seg
name: rfdetr_seg_preview-r20251104
provider: Roboflow
display_name: RF-DETR SegPreview (D)
model_path: RRFDETRSegPreviewV2_576.onnx
conf_threshold: 0.45
num_select: 300
input_width: 576      # must match the ONNX export
input_height: 576
show_boxes: false
classes:
  0: A
  1: B
  2: C
  3: D
  4: E
  5: F
```

When iporting it into X-AnyLabeling I get the following error:
`Error in loading model: exceptions must derive from BaseException`


Software version
```
App name: X-AnyLabeling
App version: 3.3.0-rc.1
Device: GPU

Operating System: Windows-11-10.0.26100-SP0
CPU: Intel64 Family 6 Model 198 Stepping 2, GenuineIntel
CPU Count: 24
RAM: 127.4 GB
Disk: 465.8/1862.4 GB
GPU: CUDA:0 (NVIDIA GeForce RTX 5090, 32607MiB)
CUDA: V12.1.66
Python Version: 3.12.12

PyQt5 Version: 5.15.11
ONNX Version: 1.19.1
ONNX Runtime Version: None
ONNX Runtime GPU Version: 1.23.2
OpenCV Contrib Python Headless Version: 4.11.0.86
```

**Comment by CVHub520** [comment]
Hi @StefanNa3Shape,

Thanks for the detailed update and for providing your export steps and YAML configuration! It's super helpful to see your process.

Regarding the issue you're encountering, could you please try specifying an **absolute path** for your `model_path` in the YAML file? Sometimes, relative paths can cause unexpected import issues, especially across different environments. You might find some relevant context or similar discussions in #441.

Let us know if that helps resolve the problem! We're here to assist further.

**Comment by Tensing2009** [comment]
Thank you for the fast response @CVHub520 !
The absolute path fixed my issue.

And for anyone interested, it was no problem importing RF-DETR SegPreview with a resolution higher than 432x432 in case you need to label with higher detail (576x576).
Also, RF-DETR SegPreview can produce masks that are overlapping, while YOLO for example has trouble doing so.
So if that is your usecase, then retrain RF-DETR SegPreview for your purpose and save yourself some time annotating.
